File: LicenseManager/Content/js/app/models/customer.js
"use strict";

var $ = require('jquery');
var _ = require('underscore');
var Backbone = require('backbone');
var Brace = require('backbone-brace');
Backbone.$ = $;

/**
 * 
 */
var CustomerModel = Brace.Model.extend({
    idAttribute: "Id",

    namedAttributes: [
        "Id",
        "Name",
        "BillingContact",
        "BillingContactEmail",
        "TechnicalContact",
        "TechnicalContactEmail",
        "IsHosted",
        "CreatedUtc"
    ]
});

module.exports = CustomerModel;

File: LicenseManager/Models/Customer.cs
using System;

namespace LicenseManager.Models
{
    public class Customer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string BillingContact { get; set; }
        public string BillingContactEmail { get; set; }
        public string TechnicalContact { get; set; }
        public string TechnicalContactEmail { get; set; }
        public bool IsHosted { get; set; }
        public DateTime CreatedUtc { get; set; }
    }
}